site/source/tutorials/007_theming/index.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: tutorial
+title: Theming
+overview: In this tutorial, you will learn how to use the Dojo 2 theming and styling system to add a theme to the sample application.
+---
+
+{% section 'first' %}
+
+# Theming
+
+## Overview
+This tutorial will extend on previous tutorials where we created the basic biz-e-corp application. In this tutorial, we will adapt the app to allow it to be themed, write a theme and apply it to our application. This will be done using the powerful theming system that is baked into Dojo 2.
```

Comment:
Not sure we should be mixing "app" and "application", perhaps better to just use "application"